package/MDAnalysis/analysis/density.py

```diff
@@ -408,8 +419,16 @@ def _prepare(self):
             # ideal solution would use images: implement 'looking across the
             # periodic boundaries' but that gets complicated when the box
             # rotates due to RMS fitting.
-            smin = np.min(coord, axis=0) - self._padding
-            smax = np.max(coord, axis=0) + self._padding
+            try:
+                smin = np.min(coord, axis=0) - self._padding
+                smax = np.max(coord, axis=0) + self._padding
+            except ValueError as err:
+                errmsg = ("No atoms in AtomGroup at input time frame. "
```

Comment:
I've mentioned this elsewhere but I think that comment got overlooked; this needs documenting in the docstring.

Reply:
I'll add it to the examples docstring of the `DensityAnalysis` class. :)

Reply:
Add it to the *Raises* section. This needs to be documented and explained.